scripts/lint

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd "$(dirname "$0")/.."
+
```

Comment:
Why not rely on pre-commit for this?

Reply:
Good point! I added the (forgotten) task to run this from VS Code in a5543ff and integrated the pre-commit hooks in 93ddaa0. Linting is much better now :)

Currently, the hooks only run via the task, not automatically on pre-commit. If desired, it can be easily added by running `pre-commit install` in a container terminal. Installing the pre-commit script by default can break committing in certain setups. For example, running the dev container on a repository from the host OS (instead of cloning into a container volume), where the host OS does not provide the required Python environment, breaks committing on the host OS if the script is installed.